# COSTMAP PARAMS
costmap :
  ros__parameters:
    map_name : 'cafe_map_best'
    map_frame_id : 'map_frame'
    map_orientation : 0.0
    map_origin : [0.0,0.0]

# GLOBAL PLANNER PARAMS
a_star_planner :
  ros__parameters:
    planner : 'a_star'
    initial_pose : [2.5, 2.5]
    goal_pose : [0.0, 0.0]

# LOCAL PLANNER PARAMS
dwa_planner :
  ros__parameters:
    min_vel : 0.0
    max_vel : 0.22
    min_w : -2.84 # rad/s
    max_w :  2.84 # rad/s
    max_accel : 0.3
    max_w_accel : 1.00
    time_step : 0.5
    time_period : 5.0
    v_resolution : 0.05
    w_resolution : 0.05
    speed_cost_gain : 10.0
    obstacle_cost_gain : 0.6 #0.5
    goal_cost_gain : 1.5 #1.0